elk/evaluation/evaluate.py

```diff
@@ -38,17 +41,18 @@ class Eval(Serializable):
 
     def execute(self):
         transfer_eval = elk_reporter_dir() / self.source / "transfer_eval"
-        cols = ["layer", "loss", "acc", "cal_acc", "auroc"]
 
-        run = Evaluate(cfg=self, eval_headers=cols, out_dir=transfer_eval)
+        run = Evaluate(cfg=self, out_dir=transfer_eval)
         run.evaluate()
 
 
 @dataclass
 class Evaluate(Run):
     cfg: Eval
 
-    def evaluate_reporter(self, layer: int, devices: list[str], world_size: int):
+    def evaluate_reporter(
+        self, layer: int, devices: list[str], world_size: int = 1
+    ) -> EvalLog:
```

Comment:
it now returns a dataclass instead of a list[str]. 
Which i think is nicer since
- Whoever wants to call Evaluate manually (e.g. in a notebook) gets back a dataclass rather than a list of unlabelled strings
- The responsbility of writing a CSV format is done somewhere else.